passkey-authenticator/src/authenticator/make_credential.rs

```diff
@@ -107,7 +107,10 @@ where
             rp_id: input.rp.id.clone(),
             credential_id: credential_id.into(),
             user_handle: input.options.rk.then_some(input.user.id.clone()),
-            counter: None,
+            counter: match self.make_credentials_with_signature_counter {
+                true => Some(0),
+                false => None,
+            },
```

Comment:
Small nit: This could be simplified like this, not sure if it makes it more or less readable though
```suggestion
            counter:self.make_credentials_with_signature_counter.then_some(0)
```

Reply:
I agree with your second thought, it doesn't really make it more readable, I'll leave it as-is